v3/how-to-guides/01-basics/a-pallet-integration/index.mdx

```diff
@@ -15,89 +15,89 @@ You will learn how to include a custom internal pallet that implements an event
 
 1. Create a local pallet called `pallet_something`.
 
-Import this pallet by adding the following to `/runtime/src/lib.rs`:
+  Import this pallet by adding the following to `/runtime/src/lib.rs`:
 
-```rust
-// Import your pallet.
-pub use pallet_something;
-```
+  ```rust
+  // Import your pallet.
+  pub use pallet_something;
+  ```
 
 1. Configure your pallet's runtime implementation.
    Assume the local pallet only has the `Event` and `Call` types exposed to the runtime. Add the following to `/runtime/src/lib.rs`:
 
-```rust
-// Configure your pallet.
-impl pallet_something::Config for Runtime {
-  type Event = Event;
-  type Call = Call;
-}
-```
+  ```rust
+  // Configure your pallet.
+  impl pallet_something::Config for Runtime {
+    type Event = Event;
+    type Call = Call;
+  }
+  ```
 
 1. Declare your pallet and the items it exposes.
    Include the additional `Pallet` and `Storage` types for the [runtime macro](/v3/runtime/macros/#construct_runtime). In `construct_runtime!`, add the following:
 
-```rust
-construct_runtime!(
-  pub enum Runtime where
-    Block = Block,
-    NodeBlock = opaque::Block,
-    UncheckedExtrinsic = UncheckedExtrinsic
-  {
-/* --snip-- */
-    Something: pallet_something::{Pallet, Call, Storage, Event<T>},
-/* --snip-- */
-  }
-);
-```
-
-> **NOTE:** To add an external pallet, you use a similar method to the one you used with the local pallet, but you must include all of the types your pallet exposes.
-> You must also include the relevant parameter types and constants. For examples of how to declare parameters and constants, see [`pallet_timestamp`][timestamp-frame].
+  ```rust
```

Comment:
sorry this fix was reverted :( I broke the site in #534 - I will patch this on `main` very shortly to get all the fixes in automatically - you don't need to hand do this anymore. Sorry about the trouble!

Reply:
Will be included here: https://github.com/substrate-developer-hub/substrate-docs/pull/551
you can _rebase_ on `main`:
```
git checkout main
git pull
git checkout LG/tutorial-sandbox-1
git rebase main
```
And then use an IDE to easily resolve any conflicts.

Reply:
@NukeManDan : You mean this branch isn't from latest and consequently hasn't been updated with recent fixes?